src/OpenTelemetry.Resources.Azure/AzureVmMetadataResponse.cs

```diff
@@ -77,9 +77,6 @@ internal string GetValueForField(string fieldName)
             case ResourceAttributeConstants.AzureVmScaleSetName:
                 amsValue = this.VmScaleSetName;
                 break;
-            case ResourceAttributeConstants.AzureVmSku:
-                amsValue = this.Sku;
```

Comment:
If this is being removed, then the `Sku` property can also be removed as nothing else appears to use it. The `ResourceGroupName` and `SubscriptionId` also appear to be used by nothing.